src/main/resources/application.properties

```diff
@@ -43,6 +43,7 @@ logging.level.org.hibernate.SQL = DEBUG
 logging.level.org.hibernate.type.descriptor.sql.BasicBinder = TRACE
 
 amppd.fileStorageRoot = ${AMPPD_FILE_ROOT:/tmp/amppd/}
+amppd.dropboxRoot = ${AMPPD_DROPBOX_FILE_ROOT:/tmp/amppd/dropbox}
 amppd.encryptionSecret = ${AMPPD_ENCRYPTION_SECRET:secret} 
```

Comment:
We will need to add this to application-tst.properties on potassium, and ask Brian what path we can use. As for Bamboo, it's probably OK to use the default one as long as /tmp/amppd/dropbox is created before tests run. Also each dev shall add this to their local properties. I suggest that for the future, if devs add/change properties that need local change as well, we can announce it on slack so everyone gets updated.

Reply:
Ok, sounds good.  Will do.